.github/workflows/deploy-gh-pages.yml

```
@@ -0,0 +1,24 @@
+name: Deploy to GithubPages
+
+on: pull_request
+permissions:
+  contents: write
+jobs:
+  deploy-to-gh-pages:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

лучше использовать свежие версии
```suggestion
        uses: actions/checkout@v4
```